src/main/authManager.ts

```diff
@@ -43,7 +42,10 @@ export class AuthManager {
 
     handleAppLogin = (event: Event, webContents: WebContents, request: AuthenticationResponseDetails, authInfo: AuthInfo, callback?: (username?: string, password?: string) => void) => {
         event.preventDefault();
-        const parsedURL = new URL(request.url);
+        const parsedURL = urlUtils.parseURL(request.url);
+        if (!parsedURL) {
```

Comment:
When is `parsedURL` falsey? It this changing any behaviour that the user would see?

Reply:
`parseURL` ensures that the URL passed is a valid URL. I changed this for consistency with the rest of our app. Shouldn't change any user behaviour unless the URL passed is invalid which should generate an error anyways.